requirements/AUR-CBI-MDL-0090/data-store/interfaceview.xml

```diff
@@ -152,114 +187,99 @@
     </Implementations>
 
 <ArchetypeReference archetype_library="eventlib" archetype_function="Event_generator"/>
+
```

Comment:
Add reference to Event_subscription_provider/handler/manager

Reply:
Done